components/brave_wallet/browser/solana_tx_manager_unittest.cc

```diff
@@ -766,7 +766,7 @@ TEST_F(SolanaTxManagerUnitTest, AddAndApproveTransaction) {
   ASSERT_TRUE(tx_meta3);
   EXPECT_TRUE(tx_meta3->tx()->fee_estimation());
   EXPECT_EQ(tx_meta3->tx()->fee_estimation()->base_fee, 5000U);
-  EXPECT_EQ(tx_meta3->tx()->fee_estimation()->compute_units, 69017U);
+  EXPECT_EQ(tx_meta3->tx()->fee_estimation()->compute_units, 69317U);
```

Comment:
I think it was a bug, when we use default fee per compute unit, we should still add 300 to it for priority fee instructions.

Reply:
Agree